src/uid2Sdk.ts

```diff
@@ -1,230 +1,354 @@
-import { setupGoogleTag } from './GoogleESPIntegration';
-import { Uid2ApiClient } from './uid2ApiClient';
-import { EventType, Uid2CallbackHandler, Uid2CallbackManager } from './uid2CallbackManager';
-import { UID2CookieManager } from './uid2CookieManager';
-import { Uid2Identity } from './Uid2Identity';
-import { IdentityStatus, notifyInitCallback } from './Uid2InitCallbacks';
-import { isUID2OptionsOrThrow, Uid2Options } from './Uid2Options';
-import { UID2PromiseHandler } from './uid2PromiseHandler';
-
-function hasExpired(expiry: number, now=Date.now()) {
-    return expiry <= now;
+import { Uid2ApiClient } from "./uid2ApiClient";
+import {
+  EventType,
+  Uid2CallbackHandler,
+  Uid2CallbackManager,
+} from "./uid2CallbackManager";
+import { UID2CookieManager } from "./uid2CookieManager";
+import { Uid2Identity } from "./Uid2Identity";
+import { IdentityStatus, notifyInitCallback } from "./Uid2InitCallbacks";
+import { isUID2OptionsOrThrow, Uid2Options } from "./Uid2Options";
+import { UID2PromiseHandler } from "./uid2PromiseHandler";
+
+function hasExpired(expiry: number, now = Date.now()) {
+  return expiry <= now;
 }
 
 let postUid2CreateCallback: null | (() => void) = null;
 
 export class UID2 {
-    
-    static get VERSION() {
-        return "3.0.0";
-    }
-    static get COOKIE_NAME() {
-        return "__uid_2";
+  static get VERSION() {
+    return "3.0.0";
+  }
+  static get COOKIE_NAME() {
+    return "__uid_2";
+  }
+  static get DEFAULT_REFRESH_RETRY_PERIOD_MS() {
+    return 2000;
```

Comment:
Why the change to 2000? I'm not sure where the original number came from, and 2000 seems sensible to me, but I'm curious.

Reply:
There was a conversation here: https://thetradedesk.slack.com/archives/C045X131YD9/p1670212126775819 
but I just realise we didn't put the timeout on the request itself, thus the getAdvertisingTokenAsync will still be hanging by the request, I will add timeout to the api client as well